.github/workflows/main.yml

```diff
@@ -0,0 +1,165 @@
+name: CI
+
+on: [push, pull_request]
```

Comment:
Do we need both? I don't know GH actions well, but all statuses seem duplicated on this PR.

Reply:
`pull_request` is required to build PRs from forks I think, I agree the duplicated statuses are annoying - afaict there's no option to squash these :/